datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/GraphQLEngine.java

```diff
@@ -31,6 +33,7 @@
  *
  * <p>In addition, it provides a simplified 'execute' API that accepts a 1) query string and 2) set of variables.
  */
+@Data
```

Comment:
Please don't make this a @Data class. I'd prefer everything that we expose from here to be explicit. 

Instead, we can just add `getter` methods for the information you need to retrieve

Reply:
Makes sense..Instead I can just `@Getter`. Intention is just to reduce the noise here.